adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/ModuleInitializer.cs

```diff
@@ -59,6 +59,9 @@ public static void EnsureModuleInitialized()
                 {
                     CoreLoggerBase.Default = new AdalLogger(Guid.Empty);
                     CoreTelemetryService.InitializeCoreTelemetryService(Telemetry.GetInstance() as ITelemetry);
+                    // Several statics in the library depends on platform information being timely initialized. 
+                    // The static initializer on PlatformInformationBase will ensure this gets done.
+                    new PlatformInformation();
```

Comment:
This looks like the right fix to me. I had not realized ADAL.NET was also affected.